webrender/src/resource_cache.rs

```diff
@@ -445,6 +471,23 @@ impl ResourceCache {
         }
     }
 
+    pub fn request_geometry(&mut self,
```

Comment:
we could have a method like `update_geometry_dimensions` in order to preserve the data